src/app/zap-templates/zcl/data-model/chip/time-format-localization-cluster.xml

```diff
@@ -50,25 +57,23 @@ limitations under the License.
       may have differing preferences for how dates and times are conveyed. As such, Nodes that visually
       or audibly convey time information need a mechanism by which they can be configured to use a
       user’s preferred format.</description>
-
-      <features>
-        <feature bit="0" code="CALFMT" name="CalendarFormat" summary="The Node can be configured to use different calendar formats when conveying values to a user.">
-          <optionalConform/>
-        </feature>
-      </features>
-
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <features>
+      <feature bit="0" code="CALFMT" name="CalendarFormat" summary="The Node can be configured to use different calendar formats when conveying values to a user.">
+        <optionalConform/>
+      </feature>
+    </features>
     <!-- Base data types -->
-    <attribute side="server" code="0x00" name="HourFormat" define="HOUR_FORMAT" type="HourFormatEnum" writable="true">
+    <attribute side="server" code="0x00" name="HourFormat" define="HOUR_FORMAT" type="HourFormatEnum" writable="true" min="0x00" max="0xFF">
```

Comment:
This is sort of silly, since 0x00 and 0xFF are the type min/max anyway...  Maybe check with @hasty why Alchemy is doing this?

Concretely in terms of the implementation: it uses more flash to store that pointless data.

Reply:
It's doing it because those are the max/min values of the enum (there's a 255 value for some reason), and enums skip the "are the calculated min/max values the same as the underlying type" check. Will fix in next build.